flickr.py

```diff
@@ -25,19 +25,20 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr(key, value):
```

Comment:
This is  good approach, but how do you pass multiple key value pairs? Try using a dictionary? Something like:
```
def search_flickr(additional_params_diction):
    ...
    params_diction.update(additional_params_diction)
    ...
```